src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Feedback/HTMLCSharpLanguageServerFeedbackFileLoggerProviderFactory.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Composition;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Feedback
+{
+    [Shared]
+    [Export(typeof(HTMLCSharpLanguageServerFeedbackFileLoggerProviderFactory))]
+    internal class HTMLCSharpLanguageServerFeedbackFileLoggerProviderFactory : FeedbackFileLoggerProviderFactoryBase
+    {
+        [ImportingConstructor]
```

Comment:
Why do we need this one and the one for Razor to be separate? They look identical to the untrained eye.

Reply:
> Why do we need this one and the one for Razor to be separate

Because we'd like to have two separate log files, one for HTMLCSharp Language Server & another for RZLS.

> They look identical

They essentially are. We could've equivalently just used a `DefaultLanguageServerFeedbackFileLoggerProviderFactory`, and passed in some sort of identifier for HTMLCSharp & Razor. Instead we created 2 separate classes to statically provide the identification ([per Taylor's suggestion](https://github.com/dotnet/aspnetcore-tooling/pull/2936#discussion_r556742147)). 


I'll also add a comment to address this in a future PR 😄